src/esportsbot/cogs/MusicCog.py

```diff
@@ -0,0 +1,1117 @@
+            # There are members in the channel
+            return False
+        else:
+            return True
+
+    def __play_queue(self, guild_id):
+        """
+        Starts the playback of the song at the top of the queue.
+        :param guild_id: The id of the guild the bot is playing in.
+        :return: True if the playback was successful, False otherwise.
+        """
+
+        if len(self._currently_active.get(guild_id).get('queue')) < 1:
+            # Queue is empty.. exit
+            return False
+
+        voice_client: VoiceClient = self._currently_active.get(guild_id).get('voice_client')
+
+        if voice_client.is_playing():
+            # Stop the bot if it is playing
+            voice_client.stop()
+
+        # Get the next song
+        next_song = self.__generate_link_data_from_queue(guild_id)
+        length = next_song.get('length')
+
+        self._time_allocation[guild_id] = self._time_allocation[guild_id] - length
+
+        if self._time_allocation[guild_id] < 0:
+            # If the allocated time is used up set it to 0 and exit out
+            self.__remove_active_channel(guild_id)
+            self._time_allocation[guild_id] = 0
+            return False
+
+        self._currently_active.get(guild_id)['current_song'] = next_song
+
+        before = "-reconnect 1 -reconnect_streamed 1 -reconnect_delay_max 5"
+        voice_client.play(FFmpegOpusAudio(next_song.get('stream'), before_options=before,
+                                          bitrate=int(next_song.get('bitrate')) + 10))
+        voice_client.volume = 100
+
+        return True
+
+    def __make_queue_list(self, guild_id):
+        """
+        Create a formatted string representing the queue from a server.
+        :param guild_id: The guild of the queue to turn into a string.
+        :return: A string representing the queue list.
+        """
+
+        queue_string = self._empty_queue_message
+        if len(self._currently_active.get(guild_id).get('queue')) > 30:
+            # The queue is too long to display
+            first_part = self._currently_active.get(guild_id).get('queue')[:10]
+            last_part = self._currently_active.get(guild_id).get('queue')[-10:]
+
+            extra = len(self._currently_active.get(guild_id).get('queue')) - 20
+
+            first_string = self.__song_list_to_string(first_part)
+            last_string = self.__song_list_to_string(last_part)
+
+            queue_string += f"{first_string}... and `{extra}` more \n{last_string}"
+        else:
+            queue_string += self.__song_list_to_string(self._currently_active.get(guild_id).get('queue'))
+
+        return queue_string
+
+    def __song_list_to_string(self, songs):
+        """
+        Turn a list into a string.
+        :param songs: The list of songs to turn into a string.
+        :return: A string representing a queue list.
+        """
+
+        string = ""
+        for x in range(len(songs)):
```

Comment:
Instead of:
```
        string = ""
        for x in range(len(songs)):
            index = x + 1
            item = songs[x]
            string += f"{index}. {item.get('title')}\n"

        return string
```

You could:
```
        return "\n".join(str(songNum+1) + ". " + song.get('title') for songNum, song in enumerate(songs))
```